apps/src/craft/craft.js

```diff
@@ -716,11 +716,16 @@ Craft.reportResult = function (success) {
 
   var keepPlayingText = Craft.replayTextForResult(testResultType);
 
+  const image = Craft.initialConfig.level.freePlay ?
+      Craft.gameController.getScreenshot() : null;
+  const encodedImage = image ? encodeURIComponent(image.split(',')[1]) : null;
```

Comment:
Ahh, wonder if we should provide a common utility function to do this? Or just add a comment.